src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+.body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
```

Comment:
according to [checklist ](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)point 11 every BEM block should have 1 corresponding file, please create one for card and one for stars